### PR TITLE
chore(i18n,client): processed translations

### PR DESCRIPTION
**Pull request by camperbot**
This PR was opened auto-magically by Crowdin.

### CODE REVIEW

**Review by raisedadead — APPROVED** [review]

**Review by DanielRosa74 — APPROVED** [review]
LGTM

File: client/i18n/locales/espanol/translations.json [hunk 1]
```diff
@@ -62,7 +62,8 @@
     "verify-email": "Verificar correo electrónico",
     "submit-and-go": "Enviar y pasar al siguiente desafío",
     "go-to-next": "Ir al próximo desafío",
-    "ask-later": "Pregúntame luego"
+    "ask-later": "Pregúntame luego",
+    "start-coding": "Start coding!"
```
Comment (RafaelDavisH) [inline client/i18n/locales/espanol/translations.json:66]:
```suggestion
    "start-coding": "¡Iniciar codificación!"
```

File: client/i18n/locales/espanol/translations.json [hunk 2]
```diff
@@ -288,7 +289,8 @@
       "preview": "Vista"
     },
     "help-translate": "Todavía estamos traduciendo las siguientes certificaciones.",
-    "help-translate-link": "Ayúdanos a traducir."
+    "help-translate-link": "Ayúdanos a traducir.",
+    "project-preview-title": "Here's a preview of what you will build"
```
Comment (RafaelDavisH) [inline client/i18n/locales/espanol/translations.json:293]:
```suggestion
    "project-preview-title": "Aquí hay una vista previa de lo que vas a construir"
```

**Review by RafaelDavisH — COMMENTED** [review]
I made a couple suggestions. The rest LGTM 🎉

**Pull request merged**